adserver/admin.py

```diff
@@ -210,6 +213,7 @@ class AdvertisementsInline(AdvertisementMixin, admin.TabularInline):
     model = Advertisement
 
     can_delete = False
+    list_per_page = 50  # make page load a bit faster
```

Comment:
```suggestion
```

Reply:
Yea, I meant to put it on the AdvertisementAdmin itself I think.